### PR TITLE
UXD storybook markdown files and images

### PR DESCRIPTION
**Pull request by beccaglass**
-resolved comments from last PR
-fixed markdown files
-deleted unnecessary photos from img folder

### CODE REVIEW

File: src/docs/uxd/actionsOnSystems/DOC.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Actions on systems
```
Comment (ryelo) [inline src/docs/uxd/actionsOnSystems/DOC.md:1]: New line after this line

File: src/docs/uxd/actionsOnSystems/DOC.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+For a system, there are 2 primary types of actions:
+
+- Service-specific actions: e.g. "Opt-out of vulnerability reporting"
+- Global system actions: e.g. "Update host name"
+
+## Service view of a system
```
Comment (ryelo) [inline src/docs/uxd/actionsOnSystems/DOC.md:8]: New line after this line

File: src/docs/uxd/actionsOnSystems/DOC.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Service-specific actions should be available from the view of the system from that service.
+Global system actions are not available.
+Service-specific actions can be inline, or in the header, or related to the table. (Anything that is related to the service!)
+_**Why?** Keep the actions in context of what the user is doing._
+![Service view with service-specific actions](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/serviceSpecificActions.png?raw=true)
+*Service view of a system, showing 2 service-specific actions (Remediate with Ansible, Opt-out of vulnerability reporting)*
+
+## Inventory view of a system
```
Comment (ryelo) [inline src/docs/uxd/actionsOnSystems/DOC.md:17]: New line after this line

**Review by ryelo — CHANGES_REQUESTED** [review]

**Review by katierik — APPROVED** [review]
Nice work Becca!

**Review by ryelo — APPROVED** [review]

**Comment by ryelo** [comment]
🎉

**Pull request merged**